### ISSUE TITLE
v0.9.0 fail to load refreshButton.bundle.js

### ISSUE THREAD

**Issue by Jbaal** [FredrikNoren/ungit#525]
Hi,
First I'd like to thank you for this great tool ! I recently decided to use git for my new projects, and ungit helps a lot when you're a noob at it ! :)

I just updated to v0.9.0 and now the page doesn't load correctly the refreshButton.bundle.js
![ungit fail](https://cloud.githubusercontent.com/assets/5984545/6994872/c683cd58-db28-11e4-9417-8461292951cc.jpg)

I have this error showing in the browser console
```
The key "" is not recognized and ignored.
private.jbalfonsi.net/:32 App version: 0.9.0
private.jbalfonsi.net/:34 Initing raven
http://private.jbalfonsi.net:8448/plugins/refreshButton/refreshButton.bundle.js Failed to load resource: the server responded with a status of 404 (Not Found)
private.jbalfonsi.net/:950 Google analytics inited
private.jbalfonsi.net/:957 Keen.io inited
ungit.js:29649 Uncaught Error: No component found: refreshButton
```

My config : 
node v0.12.0
npm 2.7.5
debian 7.8

Output from cli console : 
```
info: BugTracker set version: 0.9.0
options.options is deprecated. Use options.args instead.
info: BugTracker set user hash
info: BugTracker set version: 0.9.0
2015-04-04T22:13:09.003Z - info: socket.io started
2015-04-04T22:13:09.064Z - info: Listening on port 8448
Browse to http://localhost:8448/#/repository?path=%2Froot
Took 3825ms to start server.
## Ungit started ##
info: Sending to keen.io: server-start {"launchTimeMs":3825,"version":"0.9.0","userHash":"0e1eba92abd11a90fb0ecf6307fdc8bf"}
2015-04-04T22:13:09.101Z - info: BugTracker set user hash
2015-04-04T22:13:11.040Z - info: handshake authorized RMEgjS3p8rq_tBAZfqUI
2015-04-04T22:13:11.830Z - info: GET /images/logo.png
2015-04-04T22:13:11.985Z - info: GET /api/latestversion
2015-04-04T22:13:11.991Z - info: GET /api/gitversion
2015-04-04T22:13:13.490Z - info: GET /api/gitversion
2015-04-04T22:13:13.496Z - info: GET /
2015-04-04T22:13:13.500Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/app
2015-04-04T22:13:13.503Z - info: Plugin loaded: app
2015-04-04T22:13:13.504Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/commit
2015-04-04T22:13:13.506Z - info: Plugin loaded: commit
2015-04-04T22:13:13.507Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/commitdiff
2015-04-04T22:13:13.508Z - info: Plugin loaded: commitdiff
2015-04-04T22:13:13.509Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/dialogs
2015-04-04T22:13:13.511Z - info: Plugin loaded: dialogs
2015-04-04T22:13:13.511Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/gitErrors
2015-04-04T22:13:13.513Z - info: Plugin loaded: gitErrors
2015-04-04T22:13:13.513Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/graph
2015-04-04T22:13:13.515Z - info: Plugin loaded: graph
2015-04-04T22:13:13.515Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/header
2015-04-04T22:13:13.516Z - info: Plugin loaded: header
2015-04-04T22:13:13.517Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/home
2015-04-04T22:13:13.518Z - info: Plugin loaded: home
2015-04-04T22:13:13.519Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/imagediff
2015-04-04T22:13:13.520Z - info: Plugin loaded: imagediff
2015-04-04T22:13:13.521Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/login
2015-04-04T22:13:13.522Z - info: Plugin loaded: login
2015-04-04T22:13:13.523Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/path
2015-04-04T22:13:13.524Z - info: Plugin loaded: path
2015-04-04T22:13:13.525Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/progressBar
2015-04-04T22:13:13.526Z - info: Plugin loaded: progressBar
2015-04-04T22:13:13.527Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/refreshButton
2015-04-04T22:13:13.528Z - info: Plugin loaded: refreshButton
2015-04-04T22:13:13.529Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/remotes
2015-04-04T22:13:13.530Z - info: Plugin loaded: remotes
2015-04-04T22:13:13.531Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/repository
2015-04-04T22:13:13.533Z - info: Plugin loaded: repository
2015-04-04T22:13:13.533Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/sidebysidediff
2015-04-04T22:13:13.534Z - info: Plugin loaded: sidebysidediff
2015-04-04T22:13:13.535Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/staging
2015-04-04T22:13:13.537Z - info: Plugin loaded: staging
2015-04-04T22:13:13.538Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/stash
2015-04-04T22:13:13.539Z - info: Plugin loaded: stash
2015-04-04T22:13:13.540Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/submodules
2015-04-04T22:13:13.541Z - info: Plugin loaded: submodules
2015-04-04T22:13:13.542Z - info: Loading plugin: /usr/local/lib/node_modules/ungit/components/textdiff
2015-04-04T22:13:13.543Z - info: Plugin loaded: textdiff
2015-04-04T22:13:13.590Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/app
2015-04-04T22:13:13.594Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/commit
2015-04-04T22:13:13.596Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/commitdiff
2015-04-04T22:13:13.597Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/dialogs
2015-04-04T22:13:13.598Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/gitErrors
2015-04-04T22:13:13.599Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/graph
2015-04-04T22:13:13.601Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/header
2015-04-04T22:13:13.602Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/home
2015-04-04T22:13:13.603Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/imagediff
2015-04-04T22:13:13.604Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/login
2015-04-04T22:13:13.605Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/path
2015-04-04T22:13:13.607Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/progressBar
2015-04-04T22:13:13.609Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/refreshButton
2015-04-04T22:13:13.610Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/remotes
2015-04-04T22:13:13.611Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/repository
2015-04-04T22:13:13.611Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/sidebysidediff
2015-04-04T22:13:13.612Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/staging
2015-04-04T22:13:13.613Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/stash
2015-04-04T22:13:13.613Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/submodules
2015-04-04T22:13:13.614Z - info: Compiling plugin /usr/local/lib/node_modules/ungit/components/textdiff
2015-04-04T22:13:13.786Z - info: transport end (undefined)
2015-04-04T22:13:13.796Z - info: GET /api/ping
2015-04-04T22:13:13.873Z - info: GET /css/styles.css
2015-04-04T22:13:13.881Z - info: GET /serverdata.js
2015-04-04T22:13:13.900Z - info: GET /js/raven.min.js
2015-04-04T22:13:14.053Z - info: GET /plugins/commit/commit.css
2015-04-04T22:13:14.091Z - info: GET /plugins/commitdiff/commitdiff.css
2015-04-04T22:13:14.102Z - info: GET /plugins/graph/graph.css
2015-04-04T22:13:14.113Z - info: GET /plugins/header/header.css
2015-04-04T22:13:14.326Z - info: GET /plugins/home/home.css
2015-04-04T22:13:14.409Z - info: GET /plugins/imagediff/imagediff.css
2015-04-04T22:13:14.522Z - info: GET /plugins/refreshButton/refreshButton.css
2015-04-04T22:13:14.532Z - info: GET /plugins/repository/repository.css
2015-04-04T22:13:14.678Z - info: GET /plugins/sidebysidediff/sidebysidediff.css
2015-04-04T22:13:14.927Z - info: GET /plugins/staging/staging.css
2015-04-04T22:13:14.942Z - info: GET /plugins/stash/stash.css
2015-04-04T22:13:14.958Z - info: GET /plugins/textdiff/textdiff.css
2015-04-04T22:13:15.001Z - info: GET /js/ungit.js
2015-04-04T22:13:15.007Z - info: GET /plugins/app/app.bundle.js
2015-04-04T22:13:15.928Z - info: GET /plugins/commit/commit.bundle.js
2015-04-04T22:13:15.931Z - info: GET /plugins/commitdiff/commitdiff.bundle.js
2015-04-04T22:13:15.933Z - info: GET /plugins/dialogs/dialogs.bundle.js
2015-04-04T22:13:16.267Z - info: GET /plugins/graph/graph.bundle.js
2015-04-04T22:13:16.335Z - info: GET /plugins/gitErrors/gitErrors.bundle.js
2015-04-04T22:13:16.337Z - info: GET /plugins/header/header.bundle.js
2015-04-04T22:13:16.339Z - info: GET /plugins/home/home.bundle.js
2015-04-04T22:13:16.680Z - info: GET /plugins/imagediff/imagediff.bundle.js
2015-04-04T22:13:16.947Z - info: GET /plugins/login/login.bundle.js
2015-04-04T22:13:16.951Z - info: GET /plugins/progressBar/progressBar.bundle.js
2015-04-04T22:13:16.962Z - info: GET /plugins/path/path.bundle.js
2015-04-04T22:13:17.260Z - info: GET /plugins/refreshButton/refreshButton.bundle.js
2015-04-04T22:13:17.269Z - info: GET /plugins/remotes/remotes.bundle.js
2015-04-04T22:13:17.396Z - info: GET /plugins/repository/repository.bundle.js
2015-04-04T22:13:17.410Z - info: GET /plugins/sidebysidediff/sidebysidediff.bundle.js
2015-04-04T22:13:17.419Z - info: GET /plugins/staging/staging.bundle.js
2015-04-04T22:13:17.424Z - info: GET /plugins/stash/stash.bundle.js
2015-04-04T22:13:17.608Z - info: GET /plugins/submodules/submodules.bundle.js
2015-04-04T22:13:17.785Z - info: GET /plugins/textdiff/textdiff.bundle.js
2015-04-04T22:13:17.793Z - info: GET /js/raven.min.map
2015-04-04T22:13:22.388Z - info: GET /favicon.ico
```

When downgrading to 0.8.4, everything works fine !

Thx again !

**Comment by codingtwinky** [comment]
Shoot, can you do me a favor and run this for me?

1. run `npm list -g` and get the location of where ungit is installed
2. navigate to `[global ungit loc]/components/refreshbutton`
3. check if `refreshbutton.bundle.js` exist.

If it exist, I think this is a browser caching problem and clearing cache or no cache refresh.  If not, grunt failed to generate that file and we need to figure out why.

**Comment by raitucarp** [comment]
+1 here

**Comment by raitucarp** [comment]
> http://localhost:8448/plugins/refreshButton/refreshButton.bundle.js
> Cannot GET /plugins/refreshButton/refreshButton.bundle.js

**Comment by aitboudad** [comment]
probably, the generated refreshbutton.bundle.js is not correct
to make it work I renamed ```refreshbutton.bundle.js``` to ```refreshButton.bundle.js``` in ```.node_libraries/ungit/components/refreshButton/```

**Comment by Jbaal** [comment]
Hi,
```refreshbutton.bundle.js``` is indeed in the ```/components/refreshbutton``` folder, and as @aitboudad just wrote, renaming it to ```refreshButton.bundle.js``` make it work

Thx again !

**Comment by codingtwinky** [comment]
Forked [master branch](https://github.com/codingtwinky/ungit) has the potential fix.  Could you possibly try it out and confirm it is fixed so fix can be confirmed in next release?

@Jbaal @aitboudad @raitucarp

**Comment by aitboudad** [comment]
@codingtwinky can you tell us how we can install your fork :)

**Comment by codingtwinky** [comment]
You would have to install it like a dev would do, which is find (here)[https://github.com/FredrikNoren/ungit/blob/master/CONTRIBUTING.md#getting-started].

when cloning use https://github.com/codingtwinky/ungit.git.  After step 5, use the ungit as you would normally do.

**Comment by aitboudad** [comment]
@codingtwinky it's work for me, for grunt I used ```grunt --force``` to  avoid warning

```
Warning: Cannot find module 'optipng-bin' Use --force to continue.
```
thanks.

**Comment by codingtwinky** [comment]
err... you shouldn't have to use `grunt --force`.  I don't know what `optipng-bin`nor can find it in the repo so that is little concerning.

but thanks for confirming

**Comment by FredrikNoren** [comment]
Thanks for a quick bug report and a quick fix guys! This has now been released as 0.9.1

**Comment by Jbaal** [comment]
Excellent ! Thank you for the good work :)
